### PR TITLE
NCSD1454: Adds migration scripts

### PR DESCRIPTION
**Pull request by thelious1974**

### CODE REVIEW

File: create_collections.ps1 [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+        "--throughput"
+        $collection.RequestUnits
+    )
+
+    if($collection.PartitionKey) {
+        $params += "--partition-key-path"
+        $params += $collection.PartitionKey
+    }
+
+    & az $params | Out-Null
```
Comment (NickGraham101) [inline create_collections.ps1]: We use the CosmosDb PS module for managing Cosmos (https://github.com/PlagueHO/CosmosDB), is there a reason this wasn't used in this case?  (if this is a one off job then not worth spending time rewriting)

File: migrate_coursetext_collection.ps1 [hunk 2]
```diff
@@ -0,0 +1,44 @@
+param(
+    [Parameter(Mandatory=$true)]
+    [string] $DevAccountKey,
+    [Parameter(Mandatory=$true)]
+    [string] $SitAccountKey,   
+    [Parameter(Mandatory=$true)]
+    [string] $PPAccountKey
+)
+
+$dtExe = "C:\ProgramData\chocolatey\lib\azure-documentdb-data-migration-tool\tools\dt.exe"
```
Comment (NickGraham101) [inline migrate_coursetext_collection.ps1]: Path to exe should be passed in as a param with a default value of the usual path

File: migrate_pp_to_prd.ps1 [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+    "sectorsubjectareatier1s",
+    "sectorsubjectareatier2s",
+    "standards",
+    "standardsectorcodes",
+    "ukrlp",
+    "venues"
+)
+
+$sourceConnection = "AccountEndpoint=https://dfc-pp-prov-cdb.documents.azure.com:443/;AccountKey=$($SourceAccountKey);Database=providerportal"
+$targetConnection = "AccountEndpoint=https://dfc-prd-prov-cdb.documents.azure.com:443/;AccountKey=;$($TargetAccountKey);Database=providerportal"
```
Comment (NickGraham101) [inline migrate_pp_to_prd.ps1]: There's a semi-colon before the $TargetAccountKey

**Review by NickGraham101 — COMMENTED** [review]
One error to fix, the other comments aren't relevant if this is just one off work

File: create_collections.ps1 (hunk 1 above)
Comment (thelious1974) [reply create_collections.ps1]: This is pretty a one-off script for supporting next week's go-live for Course Directory

**Review by NickGraham101 — APPROVED** [review]

**Pull request merged**